hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/transaction/XACollectTransactionsMessageTask.java

```diff
@@ -96,7 +97,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return SecurityInterceptorConstants.RECOVER;
```

Comment:
collect transactions and recover? What is the relation ?

Reply:
When I trace this task and try to find the associated user API, I found that the user API is `XAResource.recover` [API](https://docs.oracle.com/javase/8/docs/api/javax/transaction/xa/XAResource.html#recover-int-) and hence I named it it this way to relate to the user API.